lambdas/get-organisations/src/index.ts

```diff
@@ -0,0 +1,39 @@
+require('dotenv').config();
+
+import Organisation from "../../../common/help-on-spot-models/dist/entity/Organisation";
+import {Database} from "../../../common/help-on-spot-models/dist/utils/Database";
+import {LambdaResponse, lambdaResponse} from "../../../common/help-on-spot-models/dist/utils/lambdaResponse";
+
+export interface LambdaInputEvent {
+    path?: string
+    pathParameters?: { organisationId: string }
+}
+
+export const handler = async (event?: LambdaInputEvent): Promise<LambdaResponse> => {
+    const database = new Database();
+    const connection = await database.getConnection();
+    let organisations: Organisation[] | undefined
+    try {
+        if (event?.pathParameters?.organisationId) {
+            organisations = await connection.getRepository(Organisation).find({
```

Comment:
In order to return an object instead of an array with an object I would use `findOne` here.